ci/jenkins/test-rancher.sh

```diff
@@ -0,0 +1,301 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+DEFAULT_WORKDIR="/var/lib/jenkins"
+WORKDIR=$DEFAULT_WORKDIR
```

Comment:
"/var/lib/jenkins" is a dir shared by all jobs and all runs. Is it better to use a different WORKDIR for each job each run? Jenkins automatically creates a dir on the Jenkins node for each job each run, and injects an env var WORKSPACE to the shell. "$WORKSPACE" expands to this dir and you can assume that one other job other run uses this dir. This avoid many conflicts when job is triggered in parallel on the same node.

Reply:
at this moment we are not supporting parallel run because currently we support only static clusters.